package NAlice.NWonderlogs;

import "mapreduce/yt/interface/protos/extension.proto";

message TPrivateUser {
    option (NYT.default_field_flags) = SERIALIZATION_YT;

    optional string Puid = 1 [json_name = "puid", (NYT.column_name) = "puid"];
    optional int64 PrivateUntilTimeMs = 2 [json_name = "private_until_time_ms", (NYT.column_name) = "private_until_time_ms"];
}
